src/os/bluestore/BitAllocator.h

```diff
@@ -246,26 +246,20 @@ class BitMapArea {
 class BitMapAreaList {
 
 private:
-  BitMapArea **m_items;
-  int64_t m_num_items;
+  std::vector<BitMapArea*> m_items;
```

Comment:
Guys, terribly sorry for lacking a comment with justification. The `BitMapAreaList` has been kept due to next part of BitMap allocator rework I will make. The idea is to have `BitMapAreaLeaf` aggregating `BitMapZones` directly (not pointers to them) while, in general, share the code with `BitMapAreaIN` and similar. Hopefully templatizing `BitMapAreaList` will provide such possibility.